tests/kernels/test_nvfp4_moe.py

```diff
@@ -0,0 +1,224 @@
+import torch
+import pytest
+from tests.kernels.utils import torch_moe
+from vllm import _custom_ops as ops
+from vllm.config import ParallelConfig, VllmConfig, set_current_vllm_config
+from vllm.model_executor.layers.fused_moe.cutlass_moe import cutlass_fp4_moe
+from vllm.model_executor.layers.fused_moe.fused_moe import fused_topk
+from vllm.platforms import current_platform
+from vllm.scalar_type import scalar_types
+
```

Comment:
Missing a CC check here
```suggestion

if not current_platform.has_device_capability(100):
    pytest.skip(reason="Nvfp4 Requires compute capability of 10 or above.",
                allow_module_level=True)

```